### DISCUSSION TITLE
Nuova versione 1.5.0

### DISCUSSION THREAD

**Discussion by trinko** [iisgiua/giuaschool#321] [Annunci]
La nuova versione contiene:
- aggiornamento a Symfony 5.4.11
- eliminazione del limite di 1 carattere per le sezioni; ora la sezione è un testo libero sino a 64 caratteri
- nuova funzionalità di passaggio al nuovo anno scolastico;
- correzione procedure di importazione utenti da file.

Maggiori dettagli sulle modifiche della versione:
[1.5.0](https://github.com/iisgiua/giuaschool/releases/tag/v1.5.0)

**Comment by sempliciniluigi89** [comment]
ma con il passaggio al nuovo anno scolastico , i genitori,attraverso il loro portale possono consultare i documenti es.le pagelle dell'anno precedente ?

**Reply by trinko** [reply]
> ma con il passaggio al nuovo anno scolastico , i genitori,attraverso il loro portale possono consultare i documenti es.le pagelle dell'anno precedente ?

I genitori e alunni vedono gli esiti e voti dell'anno precedente.
